testing/trino-product-tests/src/main/java/io/trino/tests/product/cli/TestTrinoCli.java

```diff
@@ -403,6 +403,26 @@ public void testSetRole()
         assertThat(trimLines(trino.readLinesUntilPrompt())).doesNotContain("admin");
     }
 
+    @Test(groups = CLI, timeOut = TIMEOUT)
+    public void shouldPrintExplainAnalyzePlan()
+            throws Exception
+    {
+        launchTrinoCliWithServerArgument();
+        trino.waitForPrompt();
+        trino.getProcessInput().println("EXPLAIN ANALYZE CREATE TABLE iceberg.default.test_table AS SELECT * FROM hive.default.nation LIMIT 10;");
+        List<String> lines = trimLines(trino.readLinesUntilPrompt());
+        assertThat(lines).contains("CREATE TABLE", "Query Plan");
+        trino.getProcessInput().println("EXPLAIN ANALYZE INSERT INTO iceberg.default.test_table VALUES(100, 'URUGUAY', 3, 'test comment');");
+        lines = trimLines(trino.readLinesUntilPrompt());
+        assertThat(lines).contains("INSERT", "Query Plan");
```

Comment:
```suggestion
        assertThat(lines).contains("INSERT: 1 row", "Query Plan");
```

Reply:
It is hard to do it now due to https://github.com/trinodb/trino/issues/14253, but that could be a good enhancement for the future